pkg/api/v1/types.go

```diff
@@ -1478,6 +1484,10 @@ type Container struct {
 	// Default is false.
 	// +optional
 	TTY bool `json:"tty,omitempty" protobuf:"varint,18,opt,name=tty"`
+	// List of environment variables to set in the container.
```

Comment:
This needs better explanation about priority wrt named `env` values (I assume they override this) and about ordering (I assume later entries override earlier entries), and about how configmap keys get transformed into env var names.  Keys in a configmap can hold values that env vars can not - what happens?